tdrs-backend/tdpservice/data_files/models.py

```diff
@@ -1,6 +1,7 @@
 """Define data file models."""
 import os
 
+from django.conf import settings
```

Comment:
Looks like this line could be removed?

Reply:
yea, left over from the previous solution

Reply:
```suggestion
from django.conf import settings
```